task_1/create-account.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./validationStyle.css">
+  <title>Validation form</title>
+
+</head>
+<body>
+  <div class="wrapper">
+    <div class="validation-form">
+      <span class="create-acount">Create account</span>
+      <form name="form-feilds" action="index.html" method="GET" onsubmit="return validate(event)">
+        <input type="text" class="input-feild" id="email" name="email" placeholder="Email Address" />
```

Comment:
Add a div as a container for the field 
Like 

```suggestion
<div class="form-field" > 
      <input type="text" class="input-feild" id="email" name="email" placeholder="Email Address" >
      <span .. class="error-message"> dfgdssd </span>
</div>
```